src/main/java/com/dnd/modutime/core/room/domain/Room.java

```diff
@@ -143,7 +142,10 @@ public boolean hasStartAndEndTime() {
     }
 
     public boolean includeTime(LocalTime time) {
-        return (!startTime.isAfter(time)) && (time.isBefore(endTime));
+        if (startTime.isBefore(endTime)) {
+            return (!startTime.isAfter(time)) && (time.isBefore(endTime));
```

Comment:
c: 가독성을 위해 부정조건보단 긍정조건을 사용해보면 어떨까요?
```suggestion
            return startTime.isBefore(time) && endTime.isAfter(time) && startTime.isEqual(time);
```

Reply:
저부분이 일부로 가독성을 위해서 의미 자체로 표현해주기 위해서 저렇게 표현했습니다!
저거 다 긍정으로 바꾸면 오히려 이해하기가 어려워서요

(!startTime.isAfter(time)) && (time.isBefore(endTime));
이거로 보면, time이 startTime보다 크거나같고, endTime보다 작다
이렇게 직관적으로 바로 보이거든요!

Reply:
`!startTime.isAfter(time)` 이 부분이 그대로 해석하면 startTime이 time의 이후가 아니다 라고 읽히는데,
time이 startTime보다 크거나 같다라는 말을 표현하려면 전부 time 기준으로 표현해도 괜찮을 것 같아요.
time이랑 startTime이랑 섞여서 더 헷갈리는 느낌이네요

그럼 이 방식은 어떤가요?
```suggestion
            return time.isEqual(StartTime) && time.isAfter(startTime)  && time.isBefore(endTime)
```